docsets/pyro/docset.json

```diff
@@ -1,18 +1,24 @@
 {
     "name": "pyro",
```

Comment:
You can change the name to `Pyro` here, if you want.

Reply:
I'm okay with both.